Source/WebCore/page/csp/ContentSecurityPolicy.h

```diff
@@ -240,6 +243,7 @@ class ContentSecurityPolicy {
     // We can never have both a script execution context and a ContentSecurityPolicyClient.
     ScriptExecutionContext* m_scriptExecutionContext { nullptr };
     ContentSecurityPolicyClient* m_client { nullptr };
+    ReportingClient* m_reportingClient { nullptr };
```

Comment:
Can we use CheckedPtr or WeakPtr instead?

Reply:
Yes! Done.